frontend/wrangler.toml

```
@@ -4,7 +4,7 @@ compatibility_date = "2024-09-23"
 pages_build_output_dir = "./build/client"
 
 [dev]
-port = 5102
+port = 3001

```

Did you mean:
```suggestion
port = 3000
```

Or it should be different from `vite:dev` one?

---

now that we removed the need of `SCRIPT_FRONTEND_URL`
we can have diff ports for `dev` and `preview`
just to not create confusion between scripts.
or have both scripts running in parallel